src/loaders/gtdb/parse_computed_genome_attribs.py

```diff
@@ -0,0 +1,223 @@
+                                    'pplacer_taxonomy'}
+
+NOTIFICATION = ' This argument should be consistent with the one used by the gtdb_genome_attribs_loader.py script.'
+
+
+def _locate_dir(root_dir, kbase_collection, load_ver, check_exists=False, tool=''):
+    result_dir = os.path.join(root_dir, loader_common_names.COLLECTION_DATA_DIR, kbase_collection, load_ver, tool)
+
+    if check_exists and not (os.path.exists(result_dir) and os.path.isdir(result_dir)):
+        raise ValueError(f"Result directory for computed genome attributes of "
+                         f"Kbase Collection: {kbase_collection} and Load Version: {load_ver} could not be found.")
+
+    return result_dir
+
+
+def _read_tsv_as_df(file_path, features, genome_id_col=None):
+    # Retrieve the desired fields from a TSV file and return the data in a dataframe
+
+    selected_cols = copy.deepcopy(features)
+    if genome_id_col:
+        selected_cols.add(genome_id_col)  # genome_id col is used to parse genome id
+    df = pd.read_csv(file_path, sep='\t', keep_default_na=False, usecols=selected_cols)
+
+    return df
+
+
+def _create_doc(row, kbase_collection, load_version, genome_id, selected_features, pre_fix='KBase'):
+    # Select specific columns and prepare them for import into Arango
+
+    # NOTE: The selected column names will have a prefix (the tool name) added to them to distinguish them from
+    #       the original metadata column names.
+
+    doc = init_genome_atrri_doc(kbase_collection, load_version, genome_id)
+
+    # distinguish the selected fields from the original metadata by adding a common prefix to their names
+    doc.update(row[list(selected_features)].rename(lambda x: pre_fix + '_' + x).to_dict())
+
+    return doc
+
+
+def _gtdbtk_row_to_doc(row, kbase_collection, load_version):
+    # Transforms row from GTDBTK summary file 'gtdbtk.ar53.summary.tsv' or 'gtdbtk.bac120.summary.tsv'
+    # into ArangoDB collection document
+
+    # parse genome id from the "user_genome" field.
+    # the "user_genome" field of GTDBTK summary is the name of input file
+    # (e.g. GCA_019057475.1_ASM1905747v1_genomic.fna.gz, GCF_000970085.1_ASM97008v1_genomic.fna.gz).
+    genome_id = '_'.join(row.user_genome.split('_')[:2])
+
+    doc = _create_doc(row, kbase_collection, load_version, genome_id, SELECTED_GTDBTK_SUMMARY_FEATURES,
+                      pre_fix='gtdb_tk')
+
+    return doc
+
+
+def gtdb_tk(root_dir, kbase_collection, load_ver):
+    """
+    Parse and formate result files generated by the GTDB-TK tool.
+    """
+    gtdb_tk_docs = list()
+
+    result_dir = _locate_dir(root_dir, kbase_collection, load_ver, tool='gtdb_tk')
+
+    # Get the list of directories for batches
+    batch_dirs = [d for d in os.listdir(result_dir) if os.path.isdir(os.path.join(result_dir, d))]
+
+    for batch_dir in batch_dirs:
+
+        # retrieve features from summary files
+        summary_files = ['gtdbtk.ar53.summary.tsv', 'gtdbtk.bac120.summary.tsv']
```

Comment:
Out of curiosity, given a set of assemblies or whatever, how does GDTB_tk separate them into `ar` and `bac`?

Reply:
I don't know that either.

Reply:
Per our conversation we accepted that "it just works"